js-exercises/last-index-of/lastIndexOf.js

```diff
@@ -0,0 +1,18 @@
+
+function lastIndexOf(...args) {
+  const numberToBeSearched = args[0]
+  const inputArray = args[1]
+  var index = 0
+  var lastIndexOfNumberInArray = inputArray.reduce((lastUpdatedIndex,arrayInput) =>{
```

Comment:
not sure but directly use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/lastIndexOf